persistence-cassandra-4.0/pom.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.stargate</groupId>
+    <artifactId>stargate</artifactId>
+    <version>0.0.11-SNAPSHOT</version>
+  </parent>
+  <groupId>io.stargate.db.cassandra</groupId>
+  <artifactId>persistence-cassandra-4.0</artifactId>
+  <properties>
+    <cassandra.version>4.0-beta1</cassandra.version>
+    <junit.version>5.6.2</junit.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>org.immutables</groupId>
+      <artifactId>value</artifactId>
+      <version>2.8.8</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.stargate.db</groupId>
+      <artifactId>persistence-api</artifactId>
+      <version>0.0.9</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.stargate.db</groupId>
+      <artifactId>persistence-common</artifactId>
+      <version>0.0.9</version>
+    </dependency>
+    <dependency>
+      <groupId>io.stargate.health</groupId>
+      <artifactId>health-checker</artifactId>
+      <version>0.0.9</version>
```

Comment:
We'll need to pull in @jorgebay's change once this is merged
https://github.com/stargate/persistence-extensions/pull/20